xml/System.Security.Cryptography/ECDiffieHellman.xml

```diff
@@ -306,7 +306,20 @@
  ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentException">
-          <paramref name="otherPartyPublicKey" /> is over a different curve than this key.</exception>
+          the curve used by <paramref name="otherPartyPublicKey" /> has a different size than the curve from this key.
+
+-or-
+
+the <paramref name="hashAlgorithm"/> parameter does not specify a hash.
+</exception>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="otherPartyPublicKey" /> is <see langword="null"/>.</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">
+          the curve used by <paramref name="otherPartyPublicKey" /> is different than the curve from this key.
+
+-or-
+
+This instance represents a only public key.</exception>
```

Comment:
"a only public key" sounds strange. Should this be phrased differently?

Reply:
Pretty sure I meant "only a public key", then copy-pasta.